### PR TITLE
fix: update types for Dock

### PR DESCRIPTION
**Pull request by lovesh**
Signed-off-by: lovesh <lovesh.bond@gmail.com>

### CODE REVIEW

**Review by TarikGul — APPROVED** [review]
LGTM, lastly just run the `build.sh` file inside of the calc folder in order to update the wasm pkg with your latest changes to the multiplier.

**Comment by lovesh** [comment]
> LGTM, lastly just run the `build.sh` file inside of the calc folder in order to update the wasm pkg with your latest changes to the multiplier.

Done

File: calc/src/calc_fee.rs [hunk 1]
```diff
@@ -50,7 +50,8 @@ impl Multiplier {
             ("westend", v) if 31 <= v => V2(new_u128(inner)),
 
             ("dock-main-runtime", _) => V2(new_u128(inner)),
-            ("dock-testnet", _) => V2(new_u128(inner)),
```
Comment (dvdplm) [inline calc/src/calc_fee.rs:53]: Is this chain gone from the world forever?

**Review by dvdplm — APPROVED** [review]
Looks sane to me. Only question is if `dock-testnet` is definitely gone forever and nobody has any interest in its historical data ever again?

**Comment by emostov** [comment]
@TarikGul after this is merged we just need to make sure we publish the calc package. I think its a bit manual with increment version in Cargo.toml, commit/build, and then publish. Also a quick sanity check with the package linked locally if we haven't done that yet

**Comment by TarikGul** [comment]
@emostov Thanks for the reminder, yea you read my mind. cc: @jsdw I believe you are releasing sidecar next week so this is a good note. I'll get a PR to update the docs for the release notes later today. I need to go through the steps myself again, and do a quick dry run to confirm the process as its been a while.

File: calc/src/calc_fee.rs (hunk 1 above)
Comment (lovesh) [reply calc/src/calc_fee.rs:53]: Yes.

**Comment by lovesh** [comment]
Thanks for merging.

**Pull request merged**